storage/mysql/68-mysql.js

```diff
@@ -114,7 +114,9 @@ module.exports = function(RED) {
                         //console.log("query:",msg.topic);
                         node.mydbConfig.pool.getConnection(function (err, conn) {
                             if (err) {
-                                conn.release()
+                                if (conn) {
```

Comment:
just checking (I haven't ;-) - does releasing conn then let the auto connect reconnect correctly also ?

Reply:
Don't know, I was just fixing the bug in the referenced issue which was definitely caused by `conn` being undefined at the point of failure.

Reply:
Ok, well done.

**Node-Red does not crash anymore** and when MySQL come back the Node **reconnect fine**.

The only things I noticed is the Node-Red logs continuously.
![node-red log](https://user-images.githubusercontent.com/15012296/154979116-1eda63f2-6ad3-4b69-afee-2f8f11a0d84a.png)

Reply:
That's going to be driven by your incoming message rate so it's one log for every failed query. 

I'm  tempted to say that it should log a failure if it can't connect and it's up to the user to use something like a catch node to handle the failure.